### ISSUE TITLE
Support for M1/arm64

### ISSUE THREAD

**Issue by lrasmus** [StatTag/StatTagMac#39]
StatTag should work on M1 macOS devices running:
- arm64 R binary
- Intel R binary

**Comment by lrasmus** [comment]
We will need to maintain 2 builds/distributions for macOS (similar to what we do for Windows).

When trying to compile in Xcode as a universal build, because our R framework version is only for one platform we get a compilation error.  R seems to be doing 2 separate builds, so there isn't a universal option there for us to tie into.

**Comment by ewhitley** [comment]
I don't think this will be viable since it relies on the user to know which version of R they're running. I think the approach I'd suggest is having the RCocoa library build against two versions and then we invoke whichever version of RCocoa we need in an XPC service. The app itself isn't the direct issue, it's the linking against R at runtime (not compile time). If we go down the dual build route, the user would need to then select which version of StatTag to run based on the version of R they want to run - and replace that version of StatTag if they want to go to a lower intel-only version of R. That make sense?

I have an extended issue discussion somewhere in the issue list - need to find it.

**Comment by ewhitley** [comment]
Commented in other ticket, but linking this here as well - conversation w/ Quinn at Apple

https://developer.apple.com/forums/thread/682381?login=true&page=1#680127022
